### PR TITLE
docs(browser-support): split browser support into its own docs section

### PR DESCRIPTION
**Pull request by juliemr**
@sjelin could you take a look?

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ browser.takeScreenshot().then(function(png) {
 ```
 
 The method to take a screenshot automatically on failure would depend on the type of failure.
-* For failures of entire specs (such as timeout or an expectation within the spec failed), you can add a reporter as such:
+* For failures of entire specs (such as timeout or an expectation within the spec failed), you can add a reporter as such (this is jasmine 1.3 syntax):
```
Comment (sjelin) [inline docs/faq.md]: I think this would be better as a comment inside the code block instead of as a parenthetical

**Comment by sjelin** [comment]
LGTM

File: docs/faq.md (hunk 1 above)
Comment (juliemr) [inline docs/faq.md]: Done!

**Comment by juliemr** [comment]
Thanks! Merged as ec985d2a355aa55f398c018aa5392ea296f07486

**Pull request closed without merging**